app/controllers/api/v1/packages_controller.rb

```diff
@@ -36,8 +36,12 @@ class PackagesController < Api::V1::ApiController
 
       api :GET, "/v1/packages", "get all packages for the item"
       def index
-        @packages = @packages.find(params[:ids].split(",")) if params[:ids].present?
-        render json: @packages, each_serializer: serializer, include_orders_packages: true
+        records = @packages #security
```

Comment:
no real need for `records = ` just use `@packages = @packages...` - like we do elsewhere and above.